odl/operator/default_ops.py

```diff
@@ -154,23 +215,38 @@ class IdentityOperator(ScalingOperator):
         IdentityOperator()(x) == x
     """
 
-    def __init__(self, space):
+    def __init__(self, domain, range=None):
```

Comment:
See also #1069

Reply:
Just make sure that the adjoint is correctly defined. Specifically in your case we have

<A1, 1> = 10

So we would need <1, A*1> = 10 as well, which means that A* = 10 I

Reply:
That's right. 

Question is: do we want to solve that issue here or in another PR (#1177)? The vast majority of our operators currently do not account for this, except for `RayTransform` because it's so obvious there.

Reply:
Question still out there. Largely I feel that I can fix it for simple cases but would leave the more complex ones for the automatic weighting PR.